R/fortify-map.r

```diff
@@ -97,20 +101,25 @@ map_data <- function(map, region = ".", exact = FALSE, ...) {
 #' ia <- map_data("county", "iowa")
 #' mid_range <- function(x) mean(range(x))
 #' seats <- do.call(rbind, lapply(split(ia, ia$subregion), function(d) {
-#'   data.frame(lat = mid_range(d$lat), long = mid_range(d$long))
+#'     data.frame(lat = mid_range(d$lat), long = mid_range(d$long), subregion = unique(d$subregion))
```

Comment:
Any reason for this addition? also, make sure to keep the 2-space indentation

Reply:
Yes this was a bug in the example code, `subregion` is used for the label aesthetic but wasn't a column in the data.

Reply:
Ah, great

Reply:
but the indentation should be fixed

Reply:
oops.. thanks!